docs/cse/integrations/automation-service.md

```diff
@@ -1,172 +1,263 @@
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<head>
-  <meta name="robots" content="noindex" />
-</head>
+This topic describes the Automation Service for Cloud SIEM Enterprise (CSE).
 
-<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
-
-This topic describes the capabilities of the Automation Service for Cloud SIEM Enterprise and includes instructions for its use.
+:::info Limited availability
+The Automation Service is available on a limited availability (LA) basis. If you would like the Automation Service enabled in your Cloud SIEM Enterprise environment, contact your Sumo Logic account representative.
+:::
 
 ## About the Automation Service
 
-The Automation Service for Cloud SIEM Enterprise (CSE) uses Sumo Logic Cloud SOAR capabilities to allow you to define and automate smart actions including enrichments and notifications. These actions can be automatically triggered when certain events occur in CSE, helping you to quickly investigate, understand, and react to potential security threats.
-
-You can interact with the service through Automations, which execute Playbooks. Playbooks are composed of one or more Actions with a workflow that could include parallel actions and logic steps. Actions are included with Integrations. Sumo Logic provides a number of integrations, actions, and playbooks with the service that can be customized; you can create their own custom integrations, actions and playbooks as well. More details about each are in the specific sections below.
+The Automation Service for Cloud SIEM Enterprise (CSE) uses [Cloud SOAR automation](/docs/cloud-soar/automation/) capabilities to allow you to define and automate smart actions, including enrichments and notifications. These actions can be automatically triggered when certain events occur in CSE, helping you to quickly investigate, understand, and react to potential security threats.
 
-The CSE Automation Service supports Enrichment, Notification and Custom Actions. 
-* Enrichment Actions can be used to gather additional information about an Entity or Insight, including Threat Indicators.
-* Notification Actions can be used to send notifications or update status in systems like CSE, CIP, Slack, Microsoft Teams, Jira, email, and so on.
+You can interact with the service through [automations](#automations), which execute playbooks. [Playbooks](#playbooks)  are composed of one or more [actions](#add-an-action-node-to-a-playbook) with a workflow that could include parallel actions and logic steps. Actions are included with [integrations](#integrations). Sumo Logic provides a number of integrations, actions, and playbooks with the service that you can customize. You can also create your own.
 
-Automations can be triggered automatically when an Insight is created and/or closed. Automations can also be executed manually via the CSE UI and API.
+:::note Limited availability
```

Comment:
In lines 12-14, it's a Limited availability admonition labeled `info` (blue block), but here it's labeled `note`. Maybe make them the same for consistency?

Reply:
Fixed. I changed it to `info`.